test/letters/utils/helpers.unit.spec.js

```diff
@@ -69,6 +68,10 @@ describe('Letters helpers: ', () => {
         expect(getBenefitOptionText(option, 'unavailable', true)).to.be.undefined;
       }, ['monthlyAwardAmount', 'serviceConnectedPercentage']);
     });
+    it('should include the awardEffectiveDate in the text for monthlyAward', () => {
+      const tree = getBenefitOptionText('monthlyAwardAmount', 20, true);
+      expect(tree.props.children[1].props.children[0]).to.contain('The effective date');
```

Comment:
This is really hacky. I tried lots of different solutions to get the text content of a child element of what I rendered, but I couldn't figure out a good way to do this. I think I ran into issues because what I'm rendering in `getBenefitOptionText` is not a React component, but just some React elements  (or DOM elements?). Can't figure out a good way of finding text content for an element that is lower down in a hierarchy of React/DOM elements.